README.md

```diff
@@ -237,7 +246,7 @@ fn get_entry_points() -> casper_types::EntryPoints {
     let mut entry_points = casper_types::EntryPoints::new();
     entry_points.add_entry_point(casper_types::EntryPoint::new(
         "constructor",
-        Vec::new(),
+        ::alloc::vec::Vec::new(),
```

Comment:
I'd recommend pulling in the `alloc::prelude` as this and several other places follow that idiom as it's used when `std` is enabled. [alloc::prelude](https://doc.rust-lang.org/alloc/prelude/index.html)

Reply:
I think thats coming in due to the macro expansion, and its cause of `cargo expand`